doc/platform_api/CMIS_Diagnostic_Monitoring_Overview_in_SONiC.md

```diff
@@ -0,0 +1,1685 @@
+
+    laser_config_freq            = FLOAT                            ; laser configured frequency in MHz
+    laser_curr_freq              = FLOAT                            ; laser current frequency in MHz
+    tx_config_power              = FLOAT                            ; configured tx output power in dbm
+```
+
+#### 2.1.2 Transceiver DOM threshold data
+
+The `TRANSCEIVER_DOM_THRESHOLD` table stores the threshold values for the DOM data.
+
+```plaintext
+    ; Defines Transceiver DOM threshold info for a port
+    key                          = TRANSCEIVER_DOM_THRESHOLD|ifname ; DOM threshold information for module on port
+    ; field                      = value
+    temphighalarm                = FLOAT                            ; temperature high alarm threshold in Celsius
+    temphighwarning              = FLOAT                            ; temperature high warning threshold in Celsius
+    templowalarm                 = FLOAT                            ; temperature low alarm threshold in Celsius
+    templowwarning               = FLOAT                            ; temperature low warning threshold in Celsius
+    vcchighalarm                 = FLOAT                            ; vcc high alarm threshold in V
+    vcchighwarning               = FLOAT                            ; vcc high warning threshold in V
+    vcclowalarm                  = FLOAT                            ; vcc low alarm threshold in V
+    vcclowwarning                = FLOAT                            ; vcc low warning threshold in V
+    txpowerhighalarm             = FLOAT                            ; tx power high alarm threshold in dBm
+    txpowerlowalarm              = FLOAT                            ; tx power low alarm threshold in dBm
+    txpowerhighwarning           = FLOAT                            ; tx power high warning threshold in dBm
+    txpowerlowwarning            = FLOAT                            ; tx power low alarm threshold in dBm
+    rxpowerhighalarm             = FLOAT                            ; rx power high alarm threshold in dBm
+    rxpowerlowalarm              = FLOAT                            ; rx power low alarm threshold in dBm
+    rxpowerhighwarning           = FLOAT                            ; rx power high warning threshold in dBm
+    rxpowerlowwarning            = FLOAT                            ; rx power low warning threshold in dBm
+    txbiashighalarm              = FLOAT                            ; tx bias high alarm threshold in mA
+    txbiaslowalarm               = FLOAT                            ; tx bias low alarm threshold in mA
+    txbiashighwarning            = FLOAT                            ; tx bias high warning threshold in mA
+    txbiaslowwarning             = FLOAT                            ; tx bias low warning threshold in mA
+    lasertemphighalarm           = FLOAT                            ; laser temperature high alarm threshold in Celsius
```

Comment:
Similarly to the previous comment on laser temperature, the threshold data could come from 
- page2 offset 152-159 or 160-167, for aux2 or aux3 mon
- or VDM threshold